en/components/progress.md

```diff
@@ -24,20 +24,27 @@ The Progress supports two layout types to fit the variety of use cases and layou
 The Progress can be used in one of the following preset color combinations:
 
 - **default**
-- success: utilizing the `success` theme color to show the progress
-- warn: utilizing the `warn` theme color to show the progress
 - error: utilizing the `error` theme color to show the progress
 - info: utilizing the `info` theme color to show the progress
+- success: utilizing the `success` theme color to show the progress
+- warn: utilizing the `warn` theme color to show the progress
 
 <img class="responsive-img" src="../images/progress_default.png" srcset="../images/progress_default@2x.png 2x" />
-<img class="responsive-img" src="../images/progress_success.png" srcset="../images/progress_success@2x.png 2x" />
-<img class="responsive-img" src="../images/progress_warn.png" srcset="../images/progress_warn@2x.png 2x" />
 <img class="responsive-img" src="../images/progress_error.png" srcset="../images/progress_error@2x.png 2x" />
 <img class="responsive-img" src="../images/progress_info.png" srcset="../images/progress_info@2x.png 2x" />
+<img class="responsive-img" src="../images/progress_success.png" srcset="../images/progress_success@2x.png 2x" />
+<img class="responsive-img" src="../images/progress_warn.png" srcset="../images/progress_warn@2x.png 2x" />
+
+## Progress Amount
+
+The Progress allows you to choose the progress amount between six percentage variants. But you don't have this option for the Indeterminate Circular and Indeterminate Linear Bar types.
+
+<img class="responsive-img" src="../images/progress_circular-progress-amount.png" srcset="../images/progress_circular-progress-amount@2x.png 2x" />
+<img class="responsive-img" src="../images/progress_linear-progress-amount.png" srcset="../images/progress_linear-progress-amount@2x.png 2x" />
```

Comment:
```suggestion
<img class="responsive-img" src="../images/progress_linear-progress-amount.png" srcset="../images/progress_linear-progress-amount@2x.png 2x" />

## Stripes

When using a Linear Bar layout shape for the Progress component, it will come with an additional option for Stripes over the Fill area. To remove the Stripes, simply set the `🌈 Stripes` color override to `transparent`.

@YoannaIvanova add an image instead of this text
```